independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -140,6 +140,7 @@
                             <goal>build</goal>
                             <goal>generate-code</goal>
                             <goal>generate-code-tests</goal>
+                            <goal>generate-filters</goal>
```

Comment:
If this goal is relevant only in the `native-with-agent` profile then it could be configured only in that profile.